### PR TITLE
Svelte & Svelte + Motoko + Internet Identity templates

### PR DESCRIPTION
**Pull request by olaszakos**
**Overview**
I added 2 clonable templates for Svelte developers for our upcoming page for Svelte developers.
- Svelte app in an asset canister
- Svelte app with Internet Identity, and a Motoko backend

### CODE REVIEW

File: svelte-motoko-starter/README.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+<p align="left" >
+  <img width="240"  src="./assets/logo.png">
+</p>
+
+# Svelte Dapp with Motoko & Internet Identity
+
+This repository is meant to give Svelte developers an easy on-ramp to get started with developing decentralized applications (Dapps in short) for the Internet Computer blockchain. Dapps, also known as smart contracts are specialized software that run on a blockchain.
```
Comment (rossberg) [inline svelte-motoko-starter/README.md:7]: Drive-by comment: Perhaps provide a link for Svelte.

File: svelte-motoko-starter/README.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+## Install dependencies
+
+Make sure you have [node.js](https://nodejs.org/) installed.
+
+### How to get this template
+
+To clone this template without downloading the entire repository, run the following command:
+
+```
+npx degit olaszakos/examples/svelte-motoko-starter svelte-motoko-starter
```
Comment (rossberg) [inline svelte-motoko-starter/README.md:49]: I suppose this has to be adjusted for this PR?

File: svelte-motoko-starter/README.md (hunk 1 above)
Comment (olaszakos) [reply svelte-motoko-starter/README.md]: Thanks, added a link

File: svelte-motoko-starter/README.md (hunk 2 above)
Comment (olaszakos) [reply svelte-motoko-starter/README.md]: Thank you, fixed

File: svelte-motoko-starter/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "Svelte",
+    "Canister",
+    "Internet Identity"
+  ],
+  "scripts": {
+    "build": "cd src/frontend && npm run build",
+    "prebuild": "npm run copy:types",
```
Comment (krpeacock) [inline svelte-motoko-starter/package.json:14]: Could we bump this to use `dfx generate` now that that's available?

File: svelte-motoko-starter/package.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    "dev": "cd src/frontend && npm run dev",
+    "prestart": "npm run copy:types",
+    "copy:types": "rsync -avr .dfx/$(echo ${DFX_NETWORK:-'**'})/canisters/** --exclude='assets/' --exclude='idl/' --exclude='*.wasm' --delete src/declarations"
```
Comment (krpeacock) [inline svelte-motoko-starter/package.json:17]: no longer needed now that `dfx generate` is available

File: svelte-motoko-starter/src/frontend/rollup.config.js [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+import css from "rollup-plugin-css-only";
+import replace from "@rollup/plugin-replace";
+import inject from "rollup-plugin-inject";
+import json from "@rollup/plugin-json";
+
+const production = !process.env.ROLLUP_WATCH;
+
+const path = require("path");
+
+function initCanisterIds() {
```
Comment (krpeacock) [inline svelte-motoko-starter/src/frontend/rollup.config.js:15]: There's a newer version of this in the starter webpack config, that may be a little simpler to integrate with rollup here: https://github.com/dfinity/sdk/blob/master/src/dfx/assets/new_project_node_files/webpack.config.js#L7

The new version creates a single map of all the canister ids to their capitalCase environment variable name, and requires less manual overriding from devs

File: svelte-starter/package.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "Svelte",
+    "Canister",
+    "Internet Identity"
+  ],
+  "scripts": {
+    "build": "cd src/frontend && npm run build",
+    "prebuild": "npm run copy:types",
```
Comment (krpeacock) [inline svelte-starter/package.json:14]: Same story here - `dfx generate`

**Review by krpeacock — CHANGES_REQUESTED** [review]
Some superficial changes - it would be nice to bring this up to the most up-to-date bundler config and package.json scripts from our default starter

**Comment by olaszakos** [comment]
> Some superficial changes - it would be nice to bring this up to the most up-to-date bundler config and package.json scripts from our default starter

@krpeacock where can i find the default starter?

**Review by krpeacock — APPROVED** [review]
